soccer/radio/NetworkRadio.cpp

```diff
@@ -0,0 +1,129 @@
+#include "NetworkRadio.hpp"
```

Comment:
I almost want to add a timeout to this. It feels if multiple robots connect with the same id over a period of time, there starts to be some connection issues. This can be (and should be) a different PR though

Reply:
ACK. Add a TODO(Kyle) and I'll get to it once I actually have robots.

Most of the complexity of this PR exists because we originally planned for dual radios, so it can be simplified a lot now.